src/instruments/src/SD/Common/PageTitle.scss

```diff
@@ -0,0 +1,9 @@
+@import "./definitions";
+
+#pageTitle {
+  &.PageTitle {
```

Comment:
At the moment both selectors are necessary to get enough CSS specificity. There is some "default" CSS which would override/affect this otherwise. Note that we don't have any component-level CSS scoping feature such as can be found in Angular.